data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/OrderedConsumerVerticle.java

```diff
@@ -176,6 +177,9 @@ private void recordsHandler(KafkaConsumerRecords<Object, CloudEvent> records) {
       bucket.forceAddTokens(records.size());
     }
 
+    Timestamp timestamp1 = new Timestamp(System.currentTimeMillis());
+    System.out.println("Time is: " + timestamp1.toString() + ". received records count: " + records.size());
```

Comment:
FYI, tf we create a [ConsumerRecordContext](https://github.com/knative-sandbox/eventing-kafka-broker/blob/935d918bdf0ecc693fa24c6a010f4fd7ef9a2b9f/data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/RecordDispatcherImpl.java#L153) here and pass it to the RecordDispatcher we get this latency included in the processing latency metric, that would definitely make it easier to monitor across multiple replicas.

Reply:
Yes - that's great. Thanks I'll add this in a separate PR in conjunction with new metric changes and remove it from here...